frontend/src/metabase/admin/settings/components/widgets/EmbeddingOption/EmbeddingOption.tsx

```diff
@@ -98,26 +107,41 @@ export const StaticEmbeddingOptionCard = () => {
   );
 };
 
-export function EmbeddingSdkOptionCard() {
-  const isEmbeddingEnabled = useSetting("enable-embedding");
+interface EmbeddingSdkOptionCardProps {
+  onToggle: ChangeEventHandler<HTMLInputElement>;
+}
+export function EmbeddingSdkOptionCard({
+  onToggle,
+}: EmbeddingSdkOptionCardProps) {
+  const isEmbeddingSdkEnabled = useSetting("enable-embedding-sdk");
   const isEE = PLUGIN_EMBEDDING.isEnabled();
 
   return (
     <EmbeddingOption
       icon={
         <SdkIcon
           className={cx(EmbeddingOptionStyle.icon, {
-            [EmbeddingOptionStyle.disabled]: !isEmbeddingEnabled,
+            [EmbeddingOptionStyle.disabled]: !isEmbeddingSdkEnabled,
           })}
         />
       }
       title={t`Embedding SDK for React`}
       label={t`PRO & ENTERPRISE`}
       description={t`Interactive embedding with full, granular control. Embed and style individual Metabase components in your app, and tailor the experience to each person. Allows for CSS styling, custom user flows, event subscriptions, and more. Only available with SSO via JWT.`}
     >
-      <LinkButton to={"/admin/settings/embedding-in-other-applications/sdk"}>
-        {!isEE ? t`Try it out` : t`Configure`}
-      </LinkButton>
+      <Flex align="center" w="100%">
+        <LinkButton to={"/admin/settings/embedding-in-other-applications/sdk"}>
+          {!isEE ? t`Try it out` : t`Configure`}
+        </LinkButton>
+        <Switch
```

Comment:
this is the embedding SDK card toggle.
![image](https://github.com/user-attachments/assets/0286576c-d494-4acb-8fe6-6721dfbefb37)